GitExtUtils/GitArgumentBuilder.cs

```diff
@@ -171,22 +98,40 @@ public void Add(GitConfigItem configItem)
             }
         }
 
-        /// <inheritdoc />
-        public override string ToString()
+        /// <summary>
+        /// Gets the length of the expected command.
+        /// </summary>
+        /// <returns>THe length of the expected command.</returns>
```

Comment:
I'd use `/// <inheritdoc />` here to avoid duplicating documentation that could get out of date.